Composer/packages/client/src/components/Notifications/index.ts

```diff
@@ -0,0 +1,6 @@
+// Copyright (c) Microsoft Corporation.
```

Comment:
nit: I recommend avoid using barrel files. It has complication with webpack tree shaking.